sklearn/utils/extmath.py

```diff
@@ -723,7 +750,8 @@ def _incremental_mean_and_var(X, last_mean, last_variance, last_sample_count):
     if last_variance is None:
         updated_variance = None
     else:
-        new_unnormalized_variance = np.nanvar(X, axis=0) * new_sample_count
+        new_unnormalized_variance = \
```

Comment:
We prefer line continuations to use parentheses rather than backslash where possible.

Reply:
I think I saw a backslash someone so I kind of went along with it. I'll take note of this.